Technologies/QuranAnalysis.md

```
@@ -979,5 +1047,6 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 ```
 
 **Commercial Use**: Quran Analysis code, framework, and corpora can be used in websites or applications (commercial/non-commercial) provided that the developer:
+
 - Link back to www.qurananalysis.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)